src/storage/irc_catalog.cpp

```diff
@@ -437,6 +435,11 @@ unique_ptr<Catalog> IRCatalog::Attach(StorageExtensionInfo *storage_info, Client
 		} else if (lower_name == "endpoint") {
 			attach_options.endpoint = StringUtil::Lower(entry.second.ToString());
 			StringUtil::RTrim(attach_options.endpoint, "/");
+		} else if (lower_name == "supports_stage_create") {
+			auto result = entry.second.GetValue<string>();
```

Comment:
`auto result = entry.second.DefaultCastAs(LogicalType::BOOLEAN);`

Doing a find for `false` is error prone